docs/iris/src/whitepapers/change_management.rst

```diff
@@ -0,0 +1,393 @@
+.. _change_management:
+
+Change Management in Iris from the User's perspective
+*****************************************************
+
+As Iris changes, user code will need revising from time to time to keep it
+working, or to maintain best practice.  At the very least, you are advised to
+review existing code to ensure it functions correctly with new releases.
+
+Here, we define ways to make this as easy as possible.
+
+.. include:: ../userguide/change_management_goals.txt
+
+
+Key principles you can rely on
+==============================
+
+Iris code editions are published as defined version releases, with a given
+major and minor version number in the version name, "major.minor.xxx",
+as explained in the :ref:`releases section <iris_change_releases>` below.
+
+    * Code that currently works should **still work**, and behave exactly the
```

Comment:
This is a fallacy. It is not possible to change *anything* without breaking something else. https://xkcd.com/1172/